packages/server/src/db/services.rs

```diff
@@ -38,9 +38,9 @@ impl Db {
 
     pub async fn get_filtered<T>(
         &self,
-        filter: impl Into<Bson>,
         filter_type: FilterType,
         field_to_filter: impl AsRef<str>,
+        filter: impl Into<Bson>,
```

Comment:
any reason we changed the order of the parameters here?

Reply:
Just felt clearer to me.